raw-spans-grouper/raw-spans-grouper/src/main/java/org/hypertrace/core/rawspansgrouper/RawSpansProcessor.java

```diff
@@ -197,6 +197,8 @@ public void process(Record<TraceIdentity, RawSpan> record) {
     }
 
     Event event = value.getEvent();
+    // spans whose trace is not created by ByPass flow, their trace will be created in below
```

Comment:
Note: For now, it is controlled by customer_id. But, we should check if this is a mirroring agent or singleton trace.